storybook/stories/support/StoryProvider.js

```diff
@@ -20,6 +20,7 @@ export const WALLETS = [
     assurance: WalletAssuranceModeOptions.NORMAL,
     hasPassword: false,
     passwordUpdateDate: new Date(),
+    isLegacy: true,
```

Comment:
Please, create a third wallet, named Legacy wallet or use the knobs, so we can toggle `isLegacy`